zerver/models/groups.py

```diff
@@ -52,6 +53,10 @@ class NamedUserGroup(UserGroup):  # type: ignore[django-manager-missing] # djang
     )
     name = models.CharField(max_length=MAX_NAME_LENGTH, db_column="name")
     description = models.TextField(default="", db_column="description")
+    date_created = models.DateTimeField(default=timezone_now, null=True)
```

Comment:
should I start the conversation in [#backend](https://chat.zulip.org/#narrow/stream/3-backend) ?

Reply:
You can backfill the column with correct data for groups created since 63f593620740343cb2b51c0e1c1336b60816cc45 via `RealmAuditLog`. I think we want `null` for groups created before that; I think that's how we handled it for channels. 

(I would expect Django to put `null` into the database in this migration).